boardwalk/prod.yml

```diff
@@ -21,7 +24,7 @@ services:
       FC_LAMBDA_DOMAIN: "${fc_lambda_domain}"
       FC_LAMBDA_PORT: "${fc_lambda_port}"
       DOS_DSS_SERVICE: "${dos_dss_service}"
-    image: quay.io/ucsc_cgl/azul-commons:commons-0.1.5-beta
+    image: quay.io/ucsc_cgl/azul-commons:commons-0.1.9-beta
```

Comment:
This should not be showing up as change. Can you please resolve it so it doesn't.

Reply:
We decided not to worry about this as the branch merges with `feature/commons`.